app/assets/stylesheets/application.scss

```diff
@@ -12,29 +21,152 @@
   display: none;
 }
 
-.input_checkbox input {
-  width: auto !important;
+.div-form {
+  padding-left: 1em;
+  padding-bottom: 1em;
+}
+
+.div-elem, 
+.form-elem {
+  padding: 1.5em 0;
+}
+
+.elem-name {
+  padding-right: 1em;
+  font-size: 130%;
+}
+
+.elem-container {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.elem-btn {
+  text-align: center;
 }
 
+//Navbar
+
+nav {
+  background-color: #428bca;
+  border-radius: 0.2em;
+}
+
+.nav-link {
+  color: white;
+  &:hover {
+    color: #343a40 ;
+  }
+}
+
+.navbar-brand {
+  color: #343a40 ;
+  font-weight: bold;
+  &:hover {
+    color: white;
+  }
+}
+
+//Index Page - Projects
+
+.project-name {
+  color: #343a40;
+  font-weight: bold;
+  font-size: 110%;
+}
+
+//Show Page - Resources
+
 .img-show {
-  width: 300px;
+  width: 24em;
```

Comment:
Perhaps best to do this as a `%` as you'll want it relative to your screen size?